worksheet_A/TerminalHacking.cpp

```diff
@@ -29,17 +48,37 @@ int main()
 	// Using a set for options guarantees that the elements are all different
 	while (options.size() < numberOfWords)
 	{
-		std::string word = words.getRandomWord();
+		string word = words.getRandomWord();
 		options.insert(word);
 	}
 
 	// Display all words
-	for each (std::string word in options)
+	for each (string word in options)
 	{
-		std::cout << word << std::endl;
+		cout << word << endl;
 	}
-
-	// TODO: implement the rest of the game
+	
+	// Game loop
+	while (playerLives > 0) 
+	{
+		cout << "Choose a word... ";
+		cin >> guessWord;
+		simScore = simCheck(guessWord, secret);
+		if (simScore == wordLength)
+		{
+			cout << "Correct" << endl;
+			cin >> guessWord;
+			playerLives = 0;
```

Comment:
Needs a comment saying that this causes the loop to end. Or better, use the `break` keyword.